static/src/stylesheets/module/email/_form.scss

```diff
@@ -7,6 +7,11 @@
     // adjustments, off grid
     width: calc(100% + 8px);
     margin-left: -4px;
+
+    @include mq($from: mobileLandscape, $until: tablet) {
+        margin-right: 50%;
+        width: calc(50% + 8px);
```

Comment:
careful with `calc`s, they can be expensive to process on lower powered devices since they generally (afaik) require multiple passes in layout. i'd be nervous to see them creep in a lot...